### PR TITLE
Fix cor surrogates

### PR DESCRIPTION
**Pull request by brborate**
verbose=FALSE for SL runs and slight edits

### CODE REVIEW

File: cor_surrogates/code/renv.lock [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (benkeser) [inline cor_surrogates/code/renv.lock:1]: Could you remove the `renv` files that have shown up in the `cor_surrogates` directory?

Comment (brborate) [reply cor_surrogates/code/renv.lock]: Done

File: cor_surrogates/code/.Rprofile [hunk 2]
```diff
@@ -0,0 +1 @@
+source("renv/activate.R")
```
Comment (benkeser) [inline cor_surrogates/code/.Rprofile:1]: And remove this file :)

Comment (brborate) [reply cor_surrogates/code/.Rprofile]: Sure thing done; I didn't realize how these sneaked in.

Comment (benkeser) [reply cor_surrogates/code/.Rprofile]: Yes, I've started to be a bit more careful with my `git add` commands. It's easy to sneak in `renv`-associated files.

Comment (brborate) [reply cor_surrogates/code/.Rprofile]: Thanks David; will be careful as well from my side. Back to the python thing. Will update you in a bit.

**Pull request merged**